cypress/support/types.ts

```diff
@@ -11,7 +11,15 @@ import {
 
 export type MockItemLike = Omit<ItemLike, 'createdAt'>;
 export type MockItemCategory = Omit<ItemCategory, 'createdAt' | 'creator'>;
-export type MockMember = Member & { thumbnail?: string };
+export type MockMember = Member & {
```

Comment:
I think you might want to use the `CompleteMember` type instead:
```suggestion
export type MockMember = CompleteMember & { thumbnail?: string };
```